pkg/controllers/externalsecret/externalsecret_controller_test.go

```diff
@@ -520,9 +520,11 @@ var _ = Describe("ExternalSecret controller", func() {
 				tplStaticKey: tplStaticVal,
 			},
 		}
-		tc.externalSecret.Spec.DataFrom = []esv1alpha1.ExternalSecretDataRemoteRef{
+		tc.externalSecret.Spec.DataFrom = []esv1alpha1.ExternalSecretDataFromRemoteRef{
 			{
-				Key: "datamap",
+				Extract: esv1alpha1.ExternalSecretExtract{
+					Key: "datamap",
+				},
```

Comment:
We might need to add some tests for the Find path as well, since we added some code to the externalsecret_controller.go
I was thinking of creating a `WithGetSecretAll` in line 531 and add the same Expect structure that we have in this block

Reply:
Yes, but I would open another issue/PR. I'm not familiar with the test framework used there and it could take me a bit longer. Just a thought